.envrc.example

```diff
@@ -7,3 +7,5 @@ export NEXT_JS_INFURA_KEY=<YOUR_INFURA_KEY>
 export NEXT_JS_DOMAIN_NAME=pooltogether.com
```

Comment:
This `NEXT_JS_DOMAIN_NAME` is defined on line 7 and 11. Prob want to only keep the line 11 one.